recipes/sip/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    pythonw configure.py --sysroot=$PREFIX
```

Comment:
Yeah, I get that it's somehow related to GUI apps.  Unfortunately, the doc you linked to doesn't shed much light -- it just says GUI apps "need to be run in a special way" :-/

I execute GUI python apps all the time, without using `pythonw` or `python.app`, I wonder what special powers `python.app` has.